client/src/components/Skills.jsx

```diff
@@ -26,37 +28,66 @@ export default function Skills({ skills, handleFormChange, setRegisterStep }) {
       .replace(/\s+/g, '');
 
   return (
-    <StyledFieldset>
-      <legend>Skills</legend>
-      <p className="instructions">Pick your primary skill</p>
-      <p>
+    <MUIFieldsetStyles>
+      <Typography variant="h4" component="h1">
+        {' '}
+        Skills
+      </Typography>
+      <Typography paragraph="true">
+        {' '}
         Select the skill you will practice the most at Code for Chicago. You don&apos;t have to be
         an expert in this skill.
-      </p>
+      </Typography>
+
+      <Typography variant="div" color=" #B00020">
+        <ErrorIcon variant="filled" />
+        All fields are required
+      </Typography>
+
+      <Typography paragraph="true">Choose only one</Typography>
 
-      {skillList.map((skill) => (
-        <label className="skillLabel" htmlFor={toCamelCase(skill)} key={skill}>
-          <input
+      <RadioGroup
+        aria-labelledby="demo-radio-buttons-group-label"
+        defaultValue="female"
```

Comment:
You can delete this `defaultValue` field. Also, you can either delete `aria-labelledby` entirely or replace the value with the id the label gets (so, `toCameCase(skill)`).